### PR TITLE
Complete documentation of some more classes

### PR DESCRIPTION
**Pull request by KoBeWi**
~~Sequel~~ Follow up to #34962

I'm not sure about `SpriteFrames.frames`. It looks like some legacy property that could be removed.

### CODE REVIEW

File: doc/classes/PoolByteArray.xml [hunk 1]
```diff
@@ -78,6 +78,11 @@
 			<return type="String">
 			</return>
 			<description>
+				Returns a hexadecimal representation of this array as a [String].
+				[codeblock]
+				var array = PoolByteArray([11, 46, 255])
+				print(array.hex_encode()) #prints: 0b2eff
```
Comment (akien-mga) [inline doc/classes/PoolByteArray.xml]: There should be a space after `#` (and capital letter), so `# Prints: 0b2eff`.

File: doc/classes/SpriteFrames.xml [hunk 2]
```diff
@@ -170,6 +170,7 @@
 	</methods>
 	<members>
 		<member name="frames" type="Array" setter="_set_frames" getter="_get_frames">
+			Compatibility property, always equals to an empty array.
```
Comment (akien-mga) [inline doc/classes/SpriteFrames.xml]: Seems like the setter does something, but it's something weird. It sets the frames of the `"default"` animation, if any:
```
void SpriteFrames::_set_frames(const Array &p_frames) {

	clear_all();
	Map<StringName, Anim>::Element *E = animations.find(SceneStringNames::get_singleton()->_default);
	ERR_FAIL_COND(!E);

	E->get().frames.resize(p_frames.size());
	for (int i = 0; i < E->get().frames.size(); i++)
		E->get().frames.write[i] = p_frames[i];
}
Array SpriteFrames::_get_frames() const {

	return Array();
}
```

Comment (akien-mga) [reply doc/classes/SpriteFrames.xml]: `"default"` is the default animation in an `AnimatedSprite` though, so it's probably functional in most cases.
I don't know why the getter is unimplemented though, might be an oversight?

**Review by akien-mga — APPROVED** [review]

File: doc/classes/SpriteFrames.xml (hunk 2 above)
Comment (KoBeWi) [reply doc/classes/SpriteFrames.xml]: Well, this line suggest that this method is not necessary anymore:
https://github.com/godotengine/godot/blob/4d3acbcb3a18520909ed6e36c3f9b5e681c2eb74/scene/2d/animated_sprite.cpp#L315

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**